BodoSQL/bodosql/tests/test_bodo_and_bodosql.py

```diff
@@ -39,9 +40,9 @@ def test_planner_reset(memory_leak_check):
     queries = ["SELECT 1", "SELECT COUNT(*) FROM TABLE1", "Select MAX(A) FROM TABLE1"]
     final_query = "DESCRIBE TABLE MYTABLE"
     for query in queries:
-        generator.parseQuery(query)
-        generator.resetPlanner()
-    generator.parseQuery(final_query)
+        JavaEntryPoint.parseQuery(generator, query)
```

Comment:
This is an example of what I hope will serve as the example pattern. This makes it clear we cross into Java, which would otherwise require an understanding of when a method implicitly crosses into Java.